src/main/java/com/github/lombrozo/testnames/ValidateMojo.java

```diff
@@ -8,11 +32,19 @@
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.project.MavenProject;
 
+/**
+ * The validate mojo.
+ *
+ * @since 0.1.0
+ */
 @Mojo(name = "check", defaultPhase = LifecyclePhase.VALIDATE)
-public class ValidateMojo extends AbstractMojo {
+public final class ValidateMojo extends AbstractMojo {
 
+    /**
+     * The project to validate.
+     */
     @Parameter(defaultValue = "${project}")
-    public MavenProject project;
+    private MavenProject project;
```

Comment:
@l3r8yJ I'm not sure about `private` here, since Maven can inject fields using reflection. Can you check locally?
1. Set version in maven `pom.xml`
```xml
<version>0.1.6-SNAPSHOT</version>
```
2. Run install command:
```bash
mvn clean install
```
3. Run plugin on some of your projects
```bash
mvn com.github.volodya-lombrozo:test-naming-conventions:0.1.6-SNAPSHOT:check
```

Reply:
@volodya-lombrozo tested, it's work